File: variablesCsharp/2.cs
using System;

namespace Variables
{
    internal class _2
    {
        static void Dos()
            {
                Console.WriteLine("Digite un número: ");

                int num1 = int.Parse(Console.ReadLine());

                int resultado = num1 * num1;

                Console.WriteLine("El cuadrado de su número es: " + resultado);
            }
        
    }
}

File: dart/variables/2.dart
import 'dart:io';

void main() {
  print("Calcular el cuadrado de un número");

  print("Digite un número");

  int num1 = int.parse(stdin.readLineSync()!);

  int cuadrado = num1 * num1;

  print("El resultado del cuadrado del número es igual a: ${cuadrado}");
  
}
